### PR TITLE
Remove Pet Command Takes in the Name, Type and Colour of the Pet.

### PR DESCRIPTION
**Pull request by Harry-Hopkinson**
Fix for #386 by making the void remove function take in the Pet Type and Colour as well as the Name.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,18 +29,8 @@
     "license": "MIT",
     "icon": "icon.png",
     "activationEvents": [
-        "onCommand:vscode-pets.start",
```
Comment (Harry-Hopkinson) [inline package.json:32]: Activation events are generated automatically from VsCode so do not need to be specified in the package.json.

**Comment by AndreasWintherMoen** [comment]
Adding type and color still doesn't prevent the issue though. It just makes it less likely. Why not add an id field? You don't even need something like uuid, just use a number incrementing for every new pet.

**Comment by tonybaloney** [comment]
I have had to force a bunch of changes to this repository to remove the cat assets (and previous copies). GitHub automatically closed the PR. You may need to create a new fork, before submitting future changes.

**Comment by tonybaloney** [comment]
What I recommend doing is:

1. Cloning the repo into a new directory
2. Getting the commit from the old clone and exporting it as a diff (https://stackoverflow.com/questions/6658313/how-can-i-generate-a-git-patch-for-a-specific-commit)
3. Applying the diff in the new directory, committing and submitting a new PR.

**Pull request closed without merging**